### PR TITLE
0.5.1-alpha fixes and features

### PR DESCRIPTION
**Pull request by mafineeek**
This PR contains few fixes, migrations and new features.

STILL WORKING - STATUS: **DRAFT**

### CODE REVIEW

**Comment by mafineeek** [comment]
ready for review ig

File: settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "owners": ["556218750337286145", "853232801398128640"],
+  "owners": ["556218750337286145", "853232801398128640", "833803685350998086"],
```
Comment (tovade) [inline settings.json:2]: remove your id please thanks

**Review by tovade — DISMISSED** [review]

File: settings.json (hunk 1 above)
Comment (mafineeek) [reply settings.json]: lol i forgot

**Review by tovade — APPROVED** [review]
Good

**Comment by tovade** [comment]
I'll merge when im home

**Pull request merged**